site/team/leichtathletik/spartenleitung_leichtathletik/index.html

```diff
@@ -0,0 +1,26 @@
+<div class="container distance">
+</br>
+    
+    <h2>Hey, ich bin der Steffen Heller</h2>
+    
+    <img src="">
+    
```

Comment:
No image source so we dont need this tag

Reply:
was meant as a spaceholder. Shall it still be removed?

Reply:
Yes please